third_party/2and3/boto/utils.pyi

```diff
@@ -0,0 +1,3 @@
+from typing import Text
+
+def pythonize_name(name: Text) -> Text: ...
```

Comment:
There's a lot of other code in this module. We generally shouldn't create a partial stub, because then users who use other functions from this module will get false positive errors that are hard to ignore. (If the module doesn't exist in typeshed at all, mypy's `--follow-imports` flag lets you shut up mypy.) Would you mind adding stubs for the other public APIs in `boto.utils`?

Reply:
Sure thing